### PR TITLE
Updated project template wrappers

### PR DESCRIPTION
**Pull request by SergeyRuzin**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,6 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-[1.8.5] - 2022-01-27
+[1.8.6] - 2022-02-14
```
Comment (andrey-brovko) [inline CHANGELOG.md:7]: This is not necessarily for a template.

File: src/Behavioral.Automation/Behavioral.Automation.csproj [hunk 2]
```diff
@@ -16,7 +16,7 @@ The whole automation code is divided into the following parts:
 - UI structure descriptive code
 - Supportive code</Description>
         <Copyright>Quantori Inc.</Copyright>
-        <PackageVersion>1.8.5</PackageVersion>
+        <PackageVersion>1.8.6</PackageVersion>
```
Comment (andrey-brovko) [inline src/Behavioral.Automation/Behavioral.Automation.csproj:19]: This is not necessarily for a template.

File: CHANGELOG.md (hunk 1 above)
Comment (SergeyRuzin) [reply CHANGELOG.md]: Reverted

File: Behavioral.Automation.Template/Behavioral.Automation.Template.Bindings/ElementWrappers/WebElementWrapper.cs [hunk 3]
```diff
@@ -89,33 +117,34 @@ public bool Stale
...
 
-        protected IDriverService Driver => _driverService;
+        protected IDriverService Driver { get; }
 
         private bool AriaEnabled
         {
             get
             {
-                switch (Element.GetAttribute("aria-disabled"))
+                return Element.GetAttribute("aria-disabled") switch
```
Comment (andrey-brovko) [inline Behavioral.Automation.Template/Behavioral.Automation.Template.Bindings/ElementWrappers/WebElementWrapper.cs:142]: I am not sure `aria-disabled` is common stuff.

Comment (SergeyRuzin) [reply Behavioral.Automation.Template/Behavioral.Automation.Template.Bindings/ElementWrappers/WebElementWrapper.cs]: Removed

**Review by andrey-brovko — APPROVED** [review]

File: Behavioral.Automation.Template/Behavioral.Automation.Template.Bindings/ElementWrappers/WebElementWrapper.cs [hunk 4]
```diff
@@ -5,40 +5,72 @@
 using System;
 using System.Collections.Generic;
 using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Text.RegularExpressions;
 
 namespace Behavioral.Automation.Template.Bindings.ElementWrappers
 {
-    public class WebElementWrapper : IWebElementWrapper
+     public class WebElementWrapper : IWebElementWrapper
```
Comment (vladimir-s-quantori) [inline Behavioral.Automation.Template/Behavioral.Automation.Template.Bindings/ElementWrappers/WebElementWrapper.cs:13]: Space?)

**Review by vladimir-s-quantori — APPROVED** [review]

File: Behavioral.Automation.Template/Behavioral.Automation.Template.Bindings/ElementWrappers/WebElementWrapper.cs (hunk 4 above)
Comment (SergeyRuzin) [reply Behavioral.Automation.Template/Behavioral.Automation.Template.Bindings/ElementWrappers/WebElementWrapper.cs]: Removed

**Pull request merged**